portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeGroup.java

```diff
@@ -14,17 +14,97 @@
 
 package com.liferay.portal.upgrade.v7_0_0;
 
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.model.GroupConstants;
 import com.liferay.portal.kernel.upgrade.UpgradeProcess;
 import com.liferay.portal.upgrade.v7_0_0.util.GroupTable;
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
 /**
  * @author Eudaldo Alonso
+ * @author Manuel de la Peña
  */
 public class UpgradeGroup extends UpgradeProcess {
 
+	protected void checkGlobalFriendlyURL(long companyId) throws Exception {
+		String friendlyURL = GroupConstants.GLOBAL_FRIENDLY_URL;
+
+		long globalGroupId = getGroupId(companyId, friendlyURL);
+
+		if (globalGroupId == 0) {
+			return;
+		}
+
+		long groupId = globalGroupId;
+		int increment = 1;
+
+		while (groupId > 0) {
+			friendlyURL = friendlyURL + increment;
+
+			groupId = getGroupId(companyId, friendlyURL);
+
+			increment++;
+		}
+
+		if (_log.isInfoEnabled()) {
+			_log.info(
+				"Modifying friendlyURL " + GroupConstants.GLOBAL_FRIENDLY_URL +
+					" of groupId " + globalGroupId + " to " + friendlyURL);
```

Comment:
@juliocamarero: maybe here we need to add more info about why the group's friendlyURL has been changed